cmd/skopeo/manifest_test.go

```diff
@@ -17,8 +17,8 @@ func TestManifestDigest(t *testing.T) {
 	}
 
 	// Error reading manifest
-	out, err := runSkopeo("manifest-digest", "/this/doesnt/exist")
-	assertTestFailed(t, out, err, "/this/doesnt/exist")
+	out, err := runSkopeo("manifest-digest", "/this/does/not/exist")
+	assertTestFailed(t, out, err, "/this/does/not/exist")
```

Comment:
(I must question the value of spell-checking paths and identifiers, with things like `/etc/passwd` and `creat(3)` a long-standing practice, but as long as it results in only PRs and not bots bringing extra work, it’s very nice to have.)

Reply:
(This is just an explanation of my logic, not an argument.)

It's much easier for me to apply changes globally than it is to try to add lots of filters against paths. -- In this case, it's easier for me to say "When writing a sentence, `doesnt` is wrong, therefore, don't use it anywhere" than "Sometimes `doesnt` is ok, and sometimes it isn't" -- it's a much harder programmatic rule to write. And if I'm not using a program, it means I have to spend more of my time reviewing and saying "oh, yeah, I can/should skip this instance". And if I revisit in 2 years (not uncommon), I'll spend that time again.

Also there will be comments that reference paths and people will get confused. Yes, `creat`, `passwd`, `Referer` and `umount` exist.
But, one of my recent PRs included a fix because someone wrote docs that talked about `unmount` expecting that to be a thing, where in fact the command for the specific tool is `umount` -- consistency is important -- people will reference docs from code and vice-versa, and they will search for things, and if it isn't spelled the way they're expecting, they may totally miss it (or lose considerable time trying to find it).

Reply:
Oh I’m absolutely not _complaining_, more in awe of your dedication.